### ISSUE TITLE
It's not possible to create a bridge without routes

### ISSUE THREAD

**Issue by qinqon** [nmstate/nmstate#413]
With version 0.0.7

configuration

```yaml
interfaces: 
  - name: br1
    type: linux-bridge
    state: up
    bridge:
      options:
        stp:
          enabled: false
      port: 
        - name: eth0
          stp-hairpin-mode: false
          stp-path-cost: 100
          stp-priority: 32
```

result:

```yaml
desired
=======
---
routes:
- destination: 0.0.0.0/0
  metric: -1
  next-hop-address: 192.168.66.2
  next-hop-interface: eth0
  table-id: 0

current
=======
---
routes: []

difference
==========
--- desired
+++ current
@@ -1,7 +1,2 @@
 ---
-routes:
-- destination: 0.0.0.0/0
-  metric: -1
-  next-hop-address: 192.168.66.2
-  next-hop-interface: eth0
-  table-id: 0
+routes: []

```

**Comment by cathay4t** [comment]
Jira issue created for this also:
https://nmstate.atlassian.net/browse/NMSTATE-221

**Comment by EdDev** [comment]
The project is managing issue through [Jira could](https://nmstate.atlassian.net/secure/RapidBoard.jspa?projectKey=NMSTATE).

**Comment by cathay4t** [comment]
This issue should be fixed in nmstate 0.3 release.
